emcc.py

```diff
@@ -569,7 +569,7 @@ def run(args):
     return 0
 
   if '-dumpversion' in args: # gcc's doc states "Print the compiler version [...] and don't do anything else."
-    print(shared.EMSCRIPTEN_VERSION)
+    print(EMSCRIPTEN_VERSION)
```

Comment:
I think its best to leave this fully qualified since `set_version_globals` won't effect that imported value.